build/package/circleci/orb.yml

```diff
@@ -115,6 +115,7 @@ jobs:
       - run:
           name: Find flag references
           command: |
+            source $BASH_ENV || true
```

Comment:
Doesn't CircleCI [already do this](https://circleci.com/docs/env-vars/#parameters-and-bash-environment), automatically?

> In every step, CircleCI uses bash to source BASH_ENV. This means that BASH_ENV is automatically loaded and run, allowing you to use interpolation and share environment variables across run steps.

Reply:
Only if the you use `bash` as shell and you specifically asks to use `sh`: 
https://github.com/launchdarkly/ld-find-code-refs/blob/main/build/package/circleci/orb.yml#L111